cruise-control-metrics-reporter/src/test/java/com/linkedin/kafka/cruisecontrol/metricsreporter/CruiseControlMetricsReporterTest.java

```diff
@@ -202,4 +202,15 @@ public void testUpdatingMetricsTopicConfig() throws ExecutionException, Interrup
     topicDescription = adminClient.describeTopics(Collections.singleton(TOPIC)).values().get(TOPIC).get();
     assertEquals(2, topicDescription.partitions().size());
   }
+  @Test
+  public void testGetKafkaBootstrapServersConfigure() throws ExecutionException, InterruptedException {
```

Comment:
Should we update this test to handle cases described on https://github.com/linkedin/cruise-control/pull/1246#discussion_r448047503?